test_sets/External_Consent_Status_OFF.txt

```diff
@@ -0,0 +1,31 @@
+./test_scripts/Polices/External_UCS/External_Consent_OFF/021_ATF_Policies_External_Consent_OFF_allowed_change_to_ON_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_to_RPCs_when_External_Consent_changes_OFF_ON
+./test_scripts/Polices/External_UCS/External_Consent_OFF/022_ATF_Policies_External_Consent_OFF_regist_new_app_disallowed_by_external_consent_entities_off.lua Check_that_SDL_assigns_permissions_to_new_registered_app_when_disallowed_by_external_consent_entities_off
+./test_scripts/Polices/External_UCS/External_Consent_OFF/023_ATF_Policies_External_Consent_OFF_regist_new_app_disallowed_by_external_consent_entities_on.lua Check_that_SDL_assigns_permissions_to_new_registered_app_when_disallowed_by_external_consent_entities_on
+./test_scripts/Polices/External_UCS/External_Consent_OFF/024_ATF_Policies_External_Consent_OFF_many_entities_pairs_in_one_group_01.lua Check_that_SDL_will_apply_externalConsentStatus_for_whole_functional_group_when_one_pair_externalConsentStatus_matches
+./test_scripts/Polices/External_UCS/External_Consent_OFF/025_ATF_Policies_External_Consent_OFF_many_entities_pairs_in_one_group_02.lua Check_that_SDL_will_apply_externalConsentStatus_for_whole_functional_group_when_two_pairs_externalConsentStatus_matches
+./test_scripts/Polices/External_UCS/External_Consent_OFF/026_ATF_Policies_External_Consent_OFF_many_entities_pairs_in_one_group_03.lua Check_that_SDL_will_apply_externalConsentStatus_for_whole_functional_group_when_by_one_disallowed_by_external_consent_entities_on_off_matches
+./test_scripts/Polices/External_UCS/External_Consent_OFF/027_ATF_Policies_External_Consent_OFF_many_entities_pairs_in_one_group_04.lua Check_that_SDLwill_apply_externalConsentStatus_for_whole_functional_group_when_one_the_same_disallowed_by_external_consent_entities_on_off_matches
+./test_scripts/Polices/External_UCS/External_Consent_OFF/028_ATF_Policies_External_Consent_OFF_same_notification_user_consent_allowed_externalConsentStatus_disallowed.lua Check_that_SDL_returns_USER_DISALLOWED_to_RPCs_when_externalConsentStatus_OFF_consentedFunctions_disallowed
+./test_scripts/Polices/External_UCS/External_Consent_OFF/029_ATF_Policies_External_Consent_OFF_same_notification_user_consent_disallowed_externalConsentStatus_allowed.lua Check_that_SDL_returns_USER_DISALLOWED_to_RPCs_when_externalConsentStatus_OFF_consentedFunctions_allowed
+./test_scripts/Polices/External_UCS/External_Consent_OFF/030_ATF_Policies_External_Consent_OFF_empty_consent_groups.lua Check_that_SDL_added_consent_group_and_external_consent_status_when_OnAppPermissionConsent_received_with_externalConsentStatus
+./test_scripts/Polices/External_UCS/External_Consent_OFF/031_ATF_Policies_External_Consent_missing.lua Check_that_SDL_processes_RPCs_when_externConsentStatus_missing_OnAppPermissionConsent_andexterna_consent_status_groups_omitted_in_policy_table
```

Comment:
@istoimenova, Thank you for clarifications
I think description in 021 script has to be correct, because it was hard to find appropriate req-t using it